rbd/snapshot_namespace.go

```diff
@@ -0,0 +1,39 @@
+// +build !luminous
```

Comment:
Github foolishly doesn't let you comment on commit message so I'm putting it here:
These are not macros in the sense I understand them, but constants.

Reply:
Fixed.